apps/web/hooks/useLinkResolver/useLinkResolver.tsx

```diff
@@ -65,6 +65,10 @@ export const routesTemplate = {
     is: '/stofnanir/[slug]/[subSlug]',
     en: '/en/organizations/[slug]/[subSlug]',
   },
+  organizationservices: {
+    is: '/stofnanir/[slug]/thjonusta',
+    en: '/en/organizations/[slug]/services',
+  },
```

Comment:
The order of url template string should be from most specific to least specific for correct type resolution (used e.g. in link resolver).
The order of these seems wrong to me, I think it should be:
`organizationservices`
`organizationsubpage`
`organizationpage`
`organizations`